rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-recorder-6.1.0-release-notes.adoc

```diff
@@ -0,0 +1,34 @@
+= RPA Recorder 6.1.0 Release Notes
+
+RPA Recorder is MuleSoft’s tool for recording a business process for automation.
+
+== TBD
+
+// == What's New
+include::partial$release-notes/rn-headings.adoc[tag=whatsNew,leveloffset=+1]
+
+* The new attributes selection panel enables you to fine-tune the XPath for web elements.
+* You can now capture _Mouse Over_ events using the `F2` key.
+* You can now activate or deactivate hotkeys.
+* The new *Open log file* button enables you to open the latest log file.
+* The new *Open log files directory* enables you to open the folder containing all RPA Recorder logs in your local computer.
+* RPA Recorder changed the file name schema for all files written, introducing the following changes:
+** The value `process name` changed to `process id`.
+** Timestamps now appear at the beginning of file names instead of at the end.
+* RPA Recorder now adds the remote debugging port to the start arguments, which enables external access to the browser wizard.
```

Comment:
Not sure about this is really needed to display here, will check with Alexander Susdorf